frontend/public/module/k8s/resource.js

```diff
@@ -1,34 +1,37 @@
 import * as _ from 'lodash-es';
-import {coFetchJSON} from '../../co-fetch';
-import {k8sBasePath} from './k8s';
-import {selectorToString} from './selector';
-import {WSFactory} from '../ws-factory';
-
-const getK8sAPIPath = kind => {
+import { coFetchJSON } from '../../co-fetch';
+import { k8sBasePath } from './k8s';
+import { selectorToString } from './selector';
+import { WSFactory } from '../ws-factory';
+
+/** @type {(model: K8sKind) => string} */
+const getK8sAPIPath = model => {
+  const isLegacy = _.get(model, 'apiGroup', 'core') === 'core' && model.apiVersion === 'v1';
```

Comment:
The URL should be `/api/v1/namespaces` instead of `/apis/core/v1/namespaces`. Thanks, Kubernetes!